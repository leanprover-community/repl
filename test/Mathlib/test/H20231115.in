{"cmd": "import Mathlib.Tactic.Cases"}

{"cmd": "example {x : Nat} : x + 1 > x := by sorry", "env": 0, "sorries": "individual"}

{"tactic": "induction' x with x hx", "proofState": 0}
