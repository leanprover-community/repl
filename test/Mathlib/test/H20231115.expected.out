{"env": 0}

{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 36},
   "parentDecl": "_example",
   "goals": ["x : Nat\n⊢ x + 1 > x"],
   "endPos": {"line": 1, "column": 41}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 0},
   "endPos": {"line": 1, "column": 7},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

{"proofStatus": "Incomplete: open goals remain",
 "proofState": 1,
 "goals":
 ["case zero\n⊢ 0 + 1 > 0",
  "case succ\nx : Nat\nhx : x + 1 > x\n⊢ x + 1 + 1 > x + 1"]}

